stdlib/universe/universe.flux

```diff
@@ -255,6 +254,33 @@ timedMovingAverage = (every, period, column="_value", tables=<-) =>
         |> duplicate(column: "_stop", as: "_time")
         |> window(every: inf)
 
+// Double Exponential Moving Average computes the double exponential moving averages of the `_value` column.
+// eg: A 5 point double exponential moving average would be called as such:
+// from(bucket: "telegraf/autogen"):
+//    |> range(start: -7d)
+//    |> doubleExponentialMovingAverage(n: 5)
+doubleExponentialMovingAverage = (n, tables=<-) =>
+    tables
+          |> exponentialMovingAverage(n:n)
+          |> duplicate(column:"_value", as:"ema")
+          |> exponentialMovingAverage(n:n)
+          |> map(fn: (r) => ({r with _value: 2.0*r.ema - r._value}))
+          |> drop(columns: ["ema"])
+
+doubleEMA = doubleExponentialMovingAverage
```

Comment:
Lets not have two names for the same thing, I am fine if we use the shorter names as their only names.